### PR TITLE
gopass-jsonapi: init at 1.11.1

### PR DESCRIPTION
**Pull request by maxhbr**
###### Motivation for this change

Gopass was split up into multiple packages extracting `gopass-jsonapi` (see https://github.com/gopasspw/gopass/blob/master/docs/setup.md#filling-in-passwords-from-browser). This adds the binary as an independent package. It is necessary for browser gopass integration via gopassbridge (see e.g. https://github.com/maxhbr/myconfig/blob/e3bab467f7f5c17eb9e70034c07445da6503739a/modules/services.xserver.programs.firefox.hm.nix#L12-L25) 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -5848,6 +5848,12 @@
     githubId = 35892750;
     name = "Maxine Aubrey";
   };
+  maxhbr = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:5851]: Please put this into a separate commit with the message ``maintainers: add maxhbr``.

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -5848,6 +5848,12 @@
...
+    email = "nixos@maxhbr.dev";
+    github = "maxhbr";
+    githubId = 1187050;
+    name = "Maximilian Huber";
+  }
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:5856]:
```suggestion
  };
```

File: pkgs/tools/security/gopass/jsonapi.nix [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+, fetchFromGitHub
+, installShellFiles
+, gopass
+}:
+
+buildGoModule rec {
+  pname = "gopass-jsonapi";
+  version = "1.11.1";
+
+  nativeBuildInputs = [ installShellFiles makeWrapper ];
```
Comment (SuperSandro2000) [inline pkgs/tools/security/gopass/jsonapi.nix:14]:
```suggestion
```

File: pkgs/tools/security/gopass/jsonapi.nix [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  src = fetchFromGitHub {
+    owner = "gopasspw";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "03xhza7n92xg12z83as9qdvvc0yx1qy6q0c7i4njvng594f9a8x2";
+  };
+
+  vendorSha256 = "0d4fyppsdfzvmjb0qvpnfnw0vl6z256bly7hfb0whk6rldks60wr";
+
+  subPackages = [ "." ];
```
Comment (SuperSandro2000) [inline pkgs/tools/security/gopass/jsonapi.nix:25]:
```suggestion
  subPackages = [ "." ];

  nativeBuildInputs = [ installShellFiles makeWrapper ];
```

File: pkgs/tools/security/gopass/jsonapi.nix [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+  doCheck = false;
+
+  buildFlagsArray = [ "-ldflags=-s -w -X main.version=${version} -X main.commit=${src.rev}" ];
+
+  wrapperPath = lib.makeBinPath [gopass];
```
Comment (SuperSandro2000) [inline pkgs/tools/security/gopass/jsonapi.nix:31]:
```suggestion
```

File: pkgs/tools/security/gopass/jsonapi.nix [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+  postFixup = ''
+    wrapProgram $out/bin/gopass-jsonapi --prefix PATH : "${wrapperPath}"
```
Comment (SuperSandro2000) [inline pkgs/tools/security/gopass/jsonapi.nix:34]:
```suggestion
    wrapProgram $out/bin/gopass-jsonapi --prefix PATH : "${lib.makeBinPath [ gopass ]}"
```

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (maxhbr) [reply maintainers/maintainer-list.nix:5856]: :facepalm:

**Comment by maxhbr** [comment]
@SuperSandro2000 : all comments are addressed

**Comment by maxhbr** [comment]
@SuperSandro2000 : all checks are green

File: pkgs/tools/security/gopass/jsonapi.nix [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    sha256 = "03xhza7n92xg12z83as9qdvvc0yx1qy6q0c7i4njvng594f9a8x2";
+  };
+
+  vendorSha256 = "0d4fyppsdfzvmjb0qvpnfnw0vl6z256bly7hfb0whk6rldks60wr";
+
+  subPackages = [ "." ];
+
+  nativeBuildInputs = [ installShellFiles makeWrapper ];
+
+  doCheck = false;
```
Comment (SuperSandro2000) [inline pkgs/tools/security/gopass/jsonapi.nix:27]: Can you add a comment why tests are disabled?

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 113273` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>gopass-jsonapi</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

gopass-jsonapi:
<details>

error: no-flags-array
`buildFlagsArray` is only intended to be used in Bash, not as a Nix attribute.

Near pkgs/tools/security/gopass/jsonapi.nix:29:3:

```
   |
29 |   buildFlagsArray = [ "-ldflags=-s -w -X main.version=${version} -X main.commit=${src.rev}" ];
   |   ^
```

See: https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-flags-array.md
Unused argument: `stdenv`.
Near pkgs/tools/security/gopass/jsonapi.nix:2:3:

```
  |
2 | , stdenv
  |   ^
```
</details>

File: pkgs/tools/security/gopass/jsonapi.nix [hunk 8]
```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
```
Comment (SuperSandro2000) [inline pkgs/tools/security/gopass/jsonapi.nix:2]:
```suggestion
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/tools/security/gopass/jsonapi.nix (hunk 7 above)
Comment (maxhbr) [reply pkgs/tools/security/gopass/jsonapi.nix]: is not necessary. Good catch
